inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -280,7 +297,31 @@ def simulate(
             logging.info('Plasma surface description not provided. '
                          'Using default file.')
             self.plasma_surface_filepath = os.path.join(
-                SIMSOPT_TEMPLATE_DIR, PLASMA_SURFACE_TEMPLATE_FILE_NAME)
+                SIMSOPT_TEMPLATE_DIR, PLASMA_SURFACE_TEMPLATE_FILENAME)
+
+        # Default weights for the objectives.
+        if objectives_weights is None:
+            self.objectives_weights = {
+                'squared_flux': 1,
+                'coils_length': 2e-03,
+                'mean_squared_curvature': 3e-04,
+                'arclength_variation': 5e-03,
+                'curvature': 3e-04
+            }
+
+        else:
```

Comment:
You don't need this else block. Just let the validation run on any `objectives_weights` provided by you or the user.